nixos/modules/services/web-apps/plausible.nix

```
@@ -78,9 +78,9 @@ in {
     server = {
       disableRegistration = mkOption {
         default = true;
-        type = types.bool;
+        type = with types; either bool string;
```

You can avoid having to add an explicit assertion below by using an enum here.

```suggestion
        type = types.enum [true false "invite_only"];
```

---

Good point. Didn't think of that. I have now included your suggestion and removed the explicit assertion.